src/category_theory/discrete_category.lean

```diff
@@ -139,7 +139,8 @@ let F : discrete α ⥤ (discrete α)ᵒᵖ := discrete.functor (λ x, op x) in
 begin
   refine equivalence.mk (functor.left_op F) F _ (discrete.nat_iso $ λ X, by simp [F]),
   refine nat_iso.of_components (λ X, by simp [F]) _,
-  tidy
+  intros,
+  sorry
```

Comment:
Similarly, this was `intros X Y f, exact dec_trivial` but it no longer finds the `decidable` instance.